load-test/src/main/scala/com/mainflux/loadtest/Engine.scala

```diff
@@ -0,0 +1,16 @@
+package com.mainflux.loadtest
+
+import io.gatling.app.Gatling
+import io.gatling.core.config.GatlingPropertiesBuilder
+import scala.util.Properties.envOrElse
+
+object Engine extends App {
```

Comment:
This can be dropped completely. Instead of reading env variables, you should use java opts (https://gatling.io/docs/2.3/cookbook/passing_parameters/). Also, make sure README file contains an instruction how to run specific simulation (see `-s` at https://gatling.io/docs/current/general/configuration/).